### DISCUSSION TITLE
Galera, o que vocês falam de desenvolvimento Indie?

### DISCUSSION THREAD

**Discussion by wilsonneto-swe** [frontendbr/forum#1586] [Discussão]
Ontem vi um vídeo sobre, devs que desenvolvem e monetizam seus próprios apps e sistemas, achei bem interessante...

Alguém aqui é dev asssim? Não freelance, mas sim desenvolver seus próprios projetos, na maior parte das vezes até sem equipe...

O que acham sobre? Tem materiais ou algo assim sobre como começar, como monetizar, quais as melhores estratégias e segmentos de apps ou soluções?

Apenas para troca de opiniões e gostaria de saber mais sobre este universo...

Abraços!

**Comment by victorferreira** [comment]
Ser _indie_ pode significar coisas diferentes em diferentes cenários, mas em geral acredito que seja quase que uma antítese ao modelo de startups onde desenvolver um negócio altamente escalável se o principal motivador no desenvolvimento de um produto/serviço. Originalmente, _indie_ é apenas uma abreviação de independente. É uma espécie de selo pra designar iniciativas que tentavam se desenvolver de forma destacada de uma indústria já estabelecida. As industrias de musica e de games tem exemplos muito claros disso.

Desenvolver algo de forma independente pode ser bastante recompensador, mas nem tudo são flores. Além do desafio técnico, praticamente qualquer negócio vai exigir uma dedicação grande em diversas áreas como marketing, suporte, finanças e etc. Mesmo que algumas tarefas possam ser terceirizadas, quem for responsável pelo produto ou serviço vai precisar garantir que tudo funcione bem e de forma integrada.

Como qualquer coisa na internet, a maior concentração de comunidade e de material sobre o assunto está em inglês. Acredito que o [Indie Hackers](https://www.indiehackers.com/) seja o melhor lugar pra encontrar experiencias de outras pessoas desenvolvendo produtos de forma independente.

De uma forma geral, desenvolvedores _indies_ costumam abordar nichos específicos e desenvolver produtos ou serviços que tem muito valor para um publico relativamente pequeno, mas que pode ser muito fiel também. Eu particularmente uso diversos produtos e serviços _indies_ que tem algo muito particular e que é difícil reproduzir. 

Acho que o melhor exemplo disso é o meu app de podcast favorito, o [Overcast](https://overcast.fm/podcasts). Quem desenvolveu o app se importa tanto com áudio que chegou a desenvolver uma _engine_ própria de audio e não dá pra encontrar em lugar nenhum. Pra alguém que, como eu, ouve muitas horas de podcasts todos os dias, algo como o Spotify é simplesmente impensável. Já a maior parte das pessoas usa o Spotify pra ouvir podcasts e não vê necessidade de usar algo diferente.

Agora eu fiquei curioso pra saber que apps e serviços _indie_ a galera usa por aqui.
